Libraries/Boards/MAX32690/APARD/examples.txt

```diff
@@ -0,0 +1,71 @@
+ADC
+ARM-DSP/arm_bayes_example
+ARM-DSP/arm_class_marks_example
+ARM-DSP/arm_convolution_example
+ARM-DSP/arm_dotproduct_example_f32
+ARM-DSP/arm_fft_bin_example
+ARM-DSP/arm_fir_example
+ARM-DSP/arm_graphic_equalizer_example
+ARM-DSP/arm_linear_interp_example
+ARM-DSP/arm_matrix_example
+ARM-DSP/arm_signal_converge_example
+ARM-DSP/arm_sin_cos_example
+ARM-DSP/arm_svm_example
+ARM-DSP/arm_variance_example
+Bluetooth/BLE_datc
+Bluetooth/BLE_dats
+Bluetooth/BLE_FreeRTOS
+Bluetooth/BLE_otac
+Bluetooth/BLE_otas
+Bluetooth/BLE4_ctr
+Bluetooth/BLE5_ctr
+Bluetooth/BLE_fcc
+Bluetooth/BLE_fit
+Bluetooth/BLE_mcs
+Bluetooth/BLE_otac
+Bluetooth/BLE_otas
+Bluetooth/BLE_periph
+Bluetooth/Bootloader
+Bluetooth/RF_Test
+CAN
+Coremark
+CRC
+CTB_AES
+DMA
+Dual_Core_Sync/Dual_core_sync_arm
+Dual_Core_Sync/Dual_core_sync_riscv
+EEPROM_Emulator
+Flash
+Flash_CLI
+FreeRTOSDemo
+GPIO
+HBMC
+Hello_World
+I2C
```

Comment:
Remove unsupported examples from examples.txt as well (this is what Eclipse uses to populate its dropdown)

Reply:
Removed I2C and Temp_Monitor in latest push